rubin_scheduler/scheduler/utils/utils.py

```diff
@@ -839,30 +842,37 @@ def __init__(self, nside=None, side_length=0.7):
             ]
         )
 
-    def __call__(self, ra, dec, rot_sky_pos=0.0):
+    def __call__(self, ra, dec, rotSkyPos=0.0):
```

Comment:
Isn't the underscore form what we actually want?

Reply:
yeah, `rotSkyPos` and `rotTelPos` are the two things that managed to get grandfathered in and not updated to proper snake_case because that's how they are in the `observation` array and the output database schema.